reana_cluster/config.py

```diff
@@ -36,7 +36,7 @@
 
 reana_cluster_ready_necessary_components = ['workflow-controller',
                                             'message-broker',
```

Comment:
Only `r-db` and `r-server`? Might be good to keep `reana-*` prefix for the other user-agnostic system-wide platform pods as well, for better consistency...

```
$ kubectl get pods
NAME                                   READY   STATUS    RESTARTS   AGE
cache-b44f567d6-28qd9                  1/1     Running   1          16h
db-7d8ff55b57-pfc8t                    1/1     Running   1          16h
message-broker-7c84b4f8d5-kswtj        1/1     Running   1          16h
server-69cfdf5746-jz4sz                2/2     Running   3          16h
workflow-controller-6cdcb4669c-5swvd   2/2     Running   2          16h
```

Reply:
The change is included in https://github.com/reanahub/reana/pull/234 :)